### PR TITLE
DOC-10688: Release notes for 7.0.5

### PR DESCRIPTION
**Pull request by RichardSmedley**
Revisions to @simon-dew 's https://github.com/couchbase/docs-server/pull/3020

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -6,6 +6,128 @@
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
 
+[#release-705]
+== Release 7.0.5 (December 2023)
```
Comment (simon-dew) [inline modules/release-notes/pages/relnotes.adoc]: Incorrect year. This is a typo from my original PR #3020 

```suggestion
== Release 7.0.5 (December 2022)
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Just my year typo.

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/release-notes/pages/relnotes.adoc]: Ha, I changed the wrong line the wrong way   :facepalm:

**Review by simon-dew — APPROVED** [review]

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -125,6 +125,9 @@ This section lists the notable fixes provided in this release.
 | https://issues.couchbase.com/browse/MB-54542[MB-54542^]
 | *Summary*: Backup/Restore fails when bucket name contains "." character
 
+| https://issues.couchbase.com/browse/MB-54263[MB-54263^]
+| *Summary*: Fix 7.0.4 issue where previously processed source bucket mutations were suppressed after undeploy-deploy cycle of an Eventing function.
```
Comment (simon-dew) [inline modules/release-notes/pages/relnotes.adoc]: Removing final . for consistency

```suggestion
| *Summary*: Fix 7.0.4 issue where previously processed source bucket mutations were suppressed after undeploy-deploy cycle of an Eventing function
```

**Pull request merged**